warehouse/redshift/redshift.go

```diff
@@ -145,7 +145,7 @@ func (rs *HandleT) CreateTable(tableName string, columns map[string]string) (err
 		distKeySql = `DISTSTYLE KEY DISTKEY("id")`
 	}
 	sqlStatement := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s ( %v ) %s SORTKEY("%s") `, name, ColumnsWithDataTypes(columns, ""), distKeySql, sortKeyField)
-	pkgLogger.Infof("Creating table in redshift for RS:%s : %v", rs.Warehouse.Destination.ID, sqlStatement)
+	pkgLogger.Debugf("Creating table in redshift for RS:%s : %v", rs.Warehouse.Destination.ID, sqlStatement)
```

Comment:
Instead of making the entire log as Debug. Why not leave it as it is and break the sqlStatement into another debug log.

Reply:
Although create table is something which will happen only if there are new events are flowing which will not be very frequent. So it's better to log it.